packages/framework/src/DependencyInjection/ShopsysFrameworkExtension.php

```diff
@@ -18,6 +18,7 @@ public function load(array $configs, ContainerBuilder $container)
     {
         $loader = new YamlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
         $loader->load('directories.yml');
+        $loader->load('parameters_common.yml');
```

Comment:
`parameters_common.yml` is imported in `AppKernel` yet, why is it here?

Reply:
Oh, I see... it is in different directory...  What do you think about merging this two parameters_common.yml files into one?

Reply:
Well, I don't think it's possible. The only way how to do it is to have all configuration in project itself because it may be needed to adjust the values.
And then I would like to have some default value in a framework.

Reply:
That is fair point from you. I didnt think in context what is a framework and what is a project. I undestand it now.